ckanext/unaids/theme/public/custom.css

```diff
@@ -205,3 +205,14 @@ section.resources H3{
 .resource-title{
   margin-bottom: 10px;
 }
+
+/* make footer sticky */
+.main{
+  display: flex;
+  min-height: 60vh;
```

Comment:
So this is saying make the website content (without footer) always at least 60% the height of the screen, then add the footer below that.  It therefore relies on the footer being more than 40% the height of the screen.  If someone has a particularly tall screen, the footer may still not stick to the bottom of the screen.  

This is definitely a step forwards, and I'd therefore just question the 60vh - why not 80vh?  or 90vh?  this would increase our chances of the footer always appearing at the bottom of the page for all screen sizes, at the cost of making some users scroll to see the whole footer.

What do you think? Can you give the rationale behind 60vh? Do you think it makes sense to bump the number up a bit?

Reply:
Yeah you're on the right track. There's no perfect way to make a sticky footer so you kinda need to gage a rough height you'd like to wait before placing it. I chose `60vh` after testing it with my 24" screen and it looked about right. I can't imagine anyone having anything larger or taller. The main focus is getting it looking good on everything under that size, especially mobile devices. So  yes - `60vh` was just something I arbitrarily chose after playing around with various numbers. Originally it was `100vh` which felt far too large. It's a function of how much fluff (headers, body padding, etc) we have on an empty page and what the max screen size we want to support before the footer becomes sticky. So i went with this.

Reply:
tldr; Under 60 felt annoying on smaller devices. Higher than 60 felt like there was too much white space in our `.main` body which felt broken.